targets/TARGET_STM/lp_ticker.c

```diff
@@ -145,6 +149,12 @@ void lp_ticker_init(void)
 
     __HAL_LPTIM_ENABLE_IT(&LptimHandle, LPTIM_IT_CMPM);
     HAL_LPTIM_Counter_Start(&LptimHandle, 0xFFFF);
+
+    /* Need to write a compare value in order to get LPTIM_FLAG_CMPOK in set_interrupt */
+    __HAL_LPTIM_CLEAR_FLAG(&LptimHandle, LPTIM_FLAG_CMPOK);
+    __HAL_LPTIM_COMPARE_SET(&LptimHandle, 0);
+    while (__HAL_LPTIM_GET_FLAG(&LptimHandle, LPTIM_FLAG_CMPOK) == RESET) {
```

Comment:
`init` is normally called outside critical section, so this should presumably be fine usually, but I see one critical section call in `ticker_resume`, but I don't see that used? Just wondering if there's a potential IRQ latency issue there.